mk/config.mk

```diff
@@ -1030,3 +1030,7 @@ CFG_FAULT_MITIGATION ?= y
 # compatibility with version v1.1 as the value of this variable is not
 # preserved in the TA dev-kit.
 CFG_TA_OPTEE_CORE_API_COMPAT_1_1 ?= n
+
+# Change supported HMAC key size range, from 64 to 1024.
+# Note that this violates GP requirements of HMAC size range.
```

Comment:
add "but is needed to pass AOSP Keymaster VTS tests" maybe? (and specify version, give link or any useful context, because it will help when times come to perhaps simplify things...)

Reply:
Done , Thank you